code/src/components/Checkbox.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export const Checkbox = (props) => {
+    return (
+        <>
+            <h2 tabIndex={'0'}>{props.question}</h2>
+                <label htmlFor={'checkbox'}>
+                    <input
+                        tabIndex={'0'}
+                        id={'checkbox'}
+                        type="checkbox"
```

Comment:
Shouldn't this be `{'checkbox'}`, to be consistent?

Reply:
I think these could all be simple quotes - either way - as Ylva says, you should be consistent (including consistency regarding double quotes or single quotes)